### PR TITLE
Microverse First Capstone Project

### PR DESCRIPTION
**Pull request by enoqueJonas**
This is Microverse's first capstone project.

With this website I decided to do something for the non-profit movement I and other friends started, it is a simple website with two pages.  Here's a link for a [video](https://www.loom.com/share/076a591fa4ff4e159bf9d11beddf10bb) that explains it.

### Technologies used
- Html
- CSS
- Javascript

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,45 @@
...
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+# 📝 License
+This project is [MIT](https://github.com/enoqueJonas/ProjectoMuda/blob/main/LICENSE) licensed.
+
+
```
Comment (Abenezer-Tilahun) [inline README.md]: - [ ] Your README file should be descriptive all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in a your repo. And so as stated in the project requirement The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.
- [ ] Kindly add a description about your project in the detail at the description header and at the built header you also used JavaScript please update it.  
- [ ] At the Get stared header section please add steps to clone and run the project locally using the command that must be used to clone and run project locally.
- [ ] Please add your GitHub, LinkedIn and twitter links to your username.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+            </header>
+            <div class="header-text">
+                <h1 class="title"> <span> "Hello World!" </span> Projecto Muda Solidary Christmas 2022 </h1>
+                <P class="primary-text"> A movement that believs in the values of giving and helping people in need, creating a positive change in the Mozambican society, soon the world, and changing how young people think about people in need. </P>
+                <div class="meeting-date-place">
+                    <h2 class="date"> 2022.12.24(SAT) ~ 25(SUN)</h2>
+                    <p class="place"> Maputo City Streets, Matola and more.</p>
+                </div>
+            </div>
+        </section>
```
Comment (Abenezer-Tilahun) [inline index.html:66]: - [ ] For making your project use semantic tags correctly and the HTML markup is well structured and follows a clear hierarchy, I recommend  you to use header with `header` tag and headline section with `section` tag . You used for both header and headline section with the same `section` tag, Please fix it.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+                    <div class="program-list-item-img">
+                        <img src="./img/charity.png" alt="">
+                    </div>
+                    <h3> Donate </h3>
+                    <p> Donate and help improve the lives of people in Maputo communities. </p>
+                </li>
+            </ul>
+            <button class="program-btn"> Join CC Global Summit 2022 </button>
+            <a href="#" class="see-program"> SEE THE WHOLE PROGRAM </a>
+        </section>
```
Comment (Abenezer-Tilahun) [inline index.html:109]: [OPTIONAL] I kindly suggest you to add the white border on hover for all the program articles just like the design guidelines.

Your all the program articles design look likes this :
Capture

The Design guidelines program articles look likes this :
Capture

File: index.html [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+                <div class="separator separator-features-speakers"></div>
+            </div>
+            <ul class="featured-speakers-list">
+   
+            </ul>
+            <button class="show-more-speakers">
+                <span> More </span>
+                <img src="./img/down-arrow.png" alt="">
+            </button>
+        </section>
```
Comment (Abenezer-Tilahun) [inline index.html:122]: - [ ] As stated in the project requirement the pages should look almost identical to the original design. So Kindly add transparent background image for all featured speakers just like the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and adjust background color for section.

The Design guidelines featured speakers section look likes this :
Capture

Your Project design Look likes this : 
Capture

File: index.html [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+            </div>
+            <div class="rights">
+                <div class="company">
+                    @Projecto Muda
+                </div>
+                <div class="rights-reserved">
+                    2022 Project Muda. <span> All rights reserved</span>
+                </div>
+            </div>
+        </section>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [ ] For making your project use semantic tags correctly and the HTML markup is well structured and follows a clear hierarchy, I recommend you to use for partner section `section` tag and for footer use  `footer` tag and make sure that footer contains a logo aligned to the left and text aligned to the right.

Your Footer page Look likes this : 
Capture

The Design Footer page look likes this : 
Capture

File: pages/about.html [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+        </header>
+        <div class="header-text">
+            <h1 class="title"> Projecto Muda Solidary Christmas 2022 </h1>
+            <P class="primary-text"> Projecto Muda is a movement that started back in 2018 and it aims to raise young people awareness on social problems. Project Muda believes that anyone can help and you do not need too much to do so. This movement has been able to help many families and the 2022 Edition won't be any different. </P>
+            <div class="meeting-date-place">
+                <h2 class="date"> Please contact us per Email for any further questions about CC Global Summit 2022 </h2>
+                <a href="#" class="place"> projectomuda@gmail.com</a>
+            </div>
+        </div>
+    </section>
```
Comment (Abenezer-Tilahun) [inline pages/about.html]: - [ ] For making your project use semantic tags correctly and the HTML markup is well structured and follows a clear hierarchy, I recommend you to use header with header tag and headline section with section tag . You used for both header and headline section with the same section tag, Please fix it.

File: pages/about.html [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+        </div>
+        <div class="rights">
+            <div class="company">
+                @Projecto Muda
+            </div>
+            <div class="rights-reserved">
+                2022 Projecto Muda. <span> All rights reserved</span>
+            </div>
+        </div>
+    </section>
```
Comment (Abenezer-Tilahun) [inline pages/about.html]: - [ ] For making your project use semantic tags correctly and the HTML markup is well structured and follows a clear hierarchy, I recommend you to use for partner section `section` tag and for footer use `footer` tag and make sure that footer contains a logo aligned to the left and text aligned to the right.

Your About footer page look likes this : 
Capture

The Design About page for footer section Look likes this : 
Capture

File: pages/about.html [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+                    <img src="" alt="">
+                </a>
+                <a href="" class="last" class="social-media-link"> English </a>
+            </div>
+            <nav class="nav-bar">
+                <a href="../index.html" class="logo"> <img src="../img/logo_muda.jpg" alt="" width="80px" height="70px"></a>
+                <ul class="nav-menu">
+                    <li class="nav-item nav-item-1">
+                        <a href="about.html" class="nav-link"> About</a>
+                    </li>
```
Comment (Abenezer-Tilahun) [inline pages/about.html]: - [ ] It is redundant to have an `àbout` link on the about page. I kindly suggest you to replace this `about` page link with the `home` page link.
Capture

- [ ] For  hamburger menu for about page please also try to add home page link .

Capture

File: scripts/script.js [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+//     //     </li>`;
+//     //     }
+//     // });
+//     // speakersSection.classList.toggle('active');
+//     // speakersUl.classList.toggle('active');
+//     // showMoreSpeakersBtnState = 0;
+//     // speakersUl.innerHTML = list;
+//     // speakersSection.appendChild(speakersUl);
+//   }
+// }
```
Comment (Abenezer-Tilahun) [inline scripts/script.js]: - [ ] I kindly suggest that your remove commented out code from your code-base to make your work neater.

File: pages/about.html [hunk 10]
```diff
@@ -0,0 +1,126 @@
...
+                    <div class="close-btn">
+                        <img src="../img/close.png" alt="Close burger menu">
+                    </div>
+                </ul>
+                <div class="burger-menu">
+                    <a href="#" class="burger-menu-btn">
+                        <img src="../img/menu.png" alt="Open menu" width="24px" height="17.5px">
+                    </a>
+                </div>
```
Comment (Abenezer-Tilahun) [inline pages/about.html:56]: - [ ] For About page of mobile screen size when I Clicked on hamburger menu nothing happens , it show an error on the console, Please try to fix it.
Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @enoqueJonas  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub Flow.
✔️ Professional README 
✔️ Good commit messages

almost

## Required Changes ♻️

_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/about-mobile-screen.css [hunk 11]
```diff
@@ -0,0 +1,212 @@
...
+  height: 5em;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding: 12px 0;
+
+  /* background: url("../img/rights.png");
+  background-size: 100%;
+  background-repeat: no-repeat; */
+}
```
Comment (levy002) [inline css/about-mobile-screen.css]: - [ ] Kindly make sure to remove commented code in codebase :wink:

File: css/wide-screen.css [hunk 12]
```diff
@@ -0,0 +1,286 @@
...
+
+.nav-item:hover {
+  background-color: rgb(0 0 0 / 5%);
+  border: 1.5px solid rgb(251 128 39 / 100%);
+}
+
+/* .nav-link:focus {
+    text-decoration: none;
+    color: #ef513c;
+} */
```
Comment (levy002) [inline css/wide-screen.css]: - [ ] Kindly remove the commented code in your codebase :wink:

File: css/wide-screen.css [hunk 13]
```diff
@@ -0,0 +1,286 @@
...
+  height: 13em;
+  background-color: #272a31;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  /* background: url("../img/partners.png");
+    background-size: 100%;
+    background-repeat: no-repeat; */
+}
```
Comment (levy002) [inline css/wide-screen.css]: - [ ] Kindly remove the commented code in your code base :wink:

File: css/wide-screen.css [hunk 14]
```diff
@@ -0,0 +1,286 @@
...
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 12px 0;
+
+  /* background: url("../img/rights.png");
+    background-size: 100%;
+    background-repeat: no-repeat; */
+}
```
Comment (levy002) [inline css/wide-screen.css]: - [ ] Kindly remove the commented code in your codebase :wink:

File: index.html [hunk 15]
```diff
@@ -0,0 +1,150 @@
...
+                    <p class="partner-logo-p partner-logo-p-5"> EJ Media </p>
+                </div>
+            </div>
+            <div class="rights">
+                <div class="company">
+                    @Projecto Muda
+                </div>
+                <div class="rights-reserved">
+                    2022 Project Muda. <span> All rights reserved</span>
+                </div>
```
Comment (levy002) [inline index.html:145]: - [ ] As requested by the previous reviewer, kindly make sure to wrap this container into a `footer` tag :wink: 
- [ ] I highly recommend you separate the `footer` from the `partners section` in order to follow the best HTML practices as suggested by the previous reviewer. :wink:

File: pages/about.html [hunk 16]
```diff
@@ -0,0 +1,126 @@
...
+            </div>
+        </div>
+        <div class="rights">
+            <div class="company">
+                @Projecto Muda
+            </div>
+            <div class="rights-reserved">
+                2022 Projecto Muda. <span> All rights reserved</span>
+            </div>
+        </div>
```
Comment (levy002) [inline pages/about.html:121]: - [ ] As requested by the previous reviewer, kindly make sure to wrap this container into a `footer` tag :wink: 
- [ ] I highly recommend you separate the `footer` from the `partners section` in order to follow the best HTML practices as suggested by the previous reviewer. :wink:

File: css/mobile-screen.css [hunk 17]
```diff
@@ -0,0 +1,353 @@
...
+}
+
+.program-list {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
```
Comment (levy002) [inline css/mobile-screen.css:169]: - [ ] Kindly make sure to align your program list cards correctly by leaving space ob both sides of each card in order to meet the provided design :wink: 

   - ### Your design
    image

   - ### Provided design
   image

File: css/mobile-screen.css [hunk 18]
```diff
@@ -0,0 +1,353 @@
...
+.featured-speakers-list {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  height: 33em;
+  padding: 0;
+  margin: 0;
+}
```
Comment (levy002) [inline css/mobile-screen.css:277]: - [ ] Kindly make sure to align your speaker list cards correctly by leaving space on left side of each card in order to meet the provided design :wink: 

   - ### Your design
   
   image


   - ### Provided design
 
   image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @enoqueJonas   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: by implementing the suggested changes by the previous reviewer. However, there are some issues that you still need to work on but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 15 above)
Comment (mckent05) [reply index.html:145]: - Good job following the advice of the previous reviewer 👏 👏

**Review by mckent05 — APPROVED** [review]
Hi @enoqueJonas,

# STATUS : APPROVED ✔️ ✔️  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
approved


## Highlights
- Your project looks good 👍 
- All linter checks are passing correctly ✔️ 
- Correct GitHub flow was followed 👏 

## Optional suggestions
**OPTIONAL** - The navbar in your project is causing a horizontal scroll on your site, please try to fix this issue 🙏 

Your Project:
scroll capstone



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**